docs/topics/api/addons.rst

```diff
@@ -145,6 +145,7 @@ This endpoint allows you to fetch a specific add-on by id, slug or guid.
     :>json int average_daily_users: The average number of users for the add-on per day.
     :>json object categories: Object holding the categories the add-on belongs to.
     :>json array categories[app_name]: Array holding the :ref:`category slugs <category-list>` the add-on belongs to for a given :ref:`add-on application <addon-detail-application>`. (Combine with the add-on ``type`` to determine the name of the category).
+    :>json string|null contributions_url: URL to the (external) webpage where the addon's authors collect monetary contributions, if set.
```

Comment:
It's not really necessary but we could document the list of supported payment providers that could end up in that url since afaik that list is fix. Might simplify frontend-devs to customize UI depending on payment provider (e.g, add little icon)

Reply:
We could document it but the list isn't static - Jorge is planning to expand it whenever a developer suggests a new service be supported - and the UX mocks for the frontend implementation don't include an icon, or suggest they should try to parse the url to match to a known particular service.